packages/babel-plugin-proposal-pipeline-operator/src/smartVisitor.js

```diff
@@ -4,45 +4,56 @@ const updateTopicReferenceVisitor = {
   PipelinePrimaryTopicReference(path) {
     path.replaceWith(t.cloneNode(this.topicId));
   },
+
   PipelineTopicExpression(path) {
     path.skip();
   },
 };
 
-const smartVisitor = {
+export default {
   BinaryExpression(path) {
     const { scope } = path;
     const { node } = path;
     const { operator, left, right } = node;
+
     if (operator !== "|>") return;
 
-    const placeholder = scope.generateUidIdentifierBasedOnNode(left);
```

Comment:
Q: Are the changes in this file needed for something? If not, could you revert it? :pray:

Reply:
Resolved in faa29eaa4b4add57bbaff99f601f8ca60424ba24.